owsi-core/owsi-core-examples/wicket-showcase/wicket-showcase-webapp/src/main/java/fr/openwide/core/showcase/web/application/portfolio/page/PortfolioMainPage.java

```diff
@@ -49,7 +51,7 @@ public PortfolioMainPage(PageParameters parameters) {
 		UserDataProvider userDataProvider = new UserDataProvider(searchTermModel, activeModel);
 		
 		// Porfolio
-		UserPortfolioPanel portfolioPanel = new UserPortfolioPanel("userPortfolio", userDataProvider, showcaseConfigurer.getPortfolioItemsPerPageUser());
+		UserPortfolioPanel portfolioPanel = new UserPortfolioPanel("userPortfolio", userDataProvider, ApplicationPropertyModel.of(ShowcaseWebappPropertyIds.PORTFOLIO_ITEMS_PER_PAGE_USER).getObject());
```

Comment:
Maybe we should pass a IModel<Long> to the UserPortfolioPanel. If not in 0.9, it might be worth to do it for 0.11.